src/liquid/components/ld-card/ld-card.css

```diff
@@ -0,0 +1,63 @@
+.ld-card {
+  --ld-card-bg-col: var(--ld-col-wht);
+  --ld-card-border-radius: var(--ld-br-l);
+  --ld-card-padding: var(--ld-sp-32);
+  --ld-card-padding-sm: var(--ld-sp-16);
+
+  box-sizing: border-box;
+  background-color: var(--ld-card-bg-col);
+  border-radius: var(--ld-card-border-radius);
+  display: block;
+  list-style: none;
+  padding: var(--ld-card-padding);
+  position: relative;
+
+  &::before,
+  &::after {
+    content: '';
+    border-radius: inherit;
+    box-shadow: var(--ld-shadow-stacked);
+    inset: 0;
+    pointer-events: none;
+    position: absolute;
+    transition: opacity 0.2s ease-out;
+  }
+
+  &::after {
+    opacity: 0;
+  }
+
+  &--sticky::before,
+  &--interactive-sticky::after {
+    box-shadow: var(--ld-shadow-sticky);
+  }
+  &--hover::before,
+  &--interactive-hover::after {
+    box-shadow: var(--ld-shadow-hover);
+  }
+  &--active::before,
+  &--interactive-active::after {
+    box-shadow: var(--ld-shadow-active);
+  }
+
+  &--interactive-stacked,
+  &--interactive-sticky,
+  &--interactive-hover,
+  &--interactive-active {
+    cursor: pointer;
+
```

Comment:
```suggestion
```

I'd assume that the card itself does not need the pointer cursor, as only clickable elements should have a pointer cursor. The interactive prop itself does not make the card clickable. It either needs to be wrapped by a clickable element or (parts of) the children need to be clickable. The respective interactive wrapping/child elements should take care of `cursor: pointer`.

Reply:
Although I'm not sure in which use case someone would need a seemingly interactive card without a cursor pointer, I I'm ok with delegating the responsibility of adding the cursor to the developer. Changed.